_things/thing-23.md

```diff
@@ -139,39 +134,26 @@ for everyone.
     Clearinghouse](http://dmtclearinghouse.esipfed.org "ESIP Data Management Training Clearinghouse")
 -   Free online [MOOC on Big Data
     Visualization](https://www.futurelearn.com/courses/big-data-visualisation "MOOC on big data visualisation")
--   Free online MOOC on [Big Data Measuring and predicting human
+-   Free online [MOOC on Big Data Measuring and predicting human
     behaviour](https://www.futurelearn.com/courses/big-data)
 
 Sometimes there are individual data units offered within a discipline
 degree, others have a number of electives or majors. Some courses
 specifically in Data Management are also available
 
-e.g. **Charles Sturt University**[**Graduate Certificate and Masters
+e.g. **Charles Sturt University** [**Graduate Certificate and Masters
 Specialisation in Data
-Management**](http://www.csu.edu.au/infostudies/ "Graduate Certificate and Masters Specialisation in Data Management")
+Management**](https://study.csu.edu.au/courses/library-information-studies/graduate-certificate-data-management "Graduate Certificate and Masters Specialisation in Data Management")
 
-From 2017 onwards the School of Information Studies at Charles Sturt
-University will be offering a Graduate Certificate in Data Management
-(GCDM). As practicing information professionals or researchers, formal
+The School of Information Studies at Charles Sturt University will be
```

Comment:
Will be? Pretty sure this Grad Cert is already being offered...

Reply:
Fixed the first part but didn't see that. Updated the branch.